src/test/java/uk/ac/ebi/eva/pipeline/runner/EvaPipelineJobLauncherCommandLineRunnerTest.java

```diff
@@ -139,32 +145,81 @@ public void genotypedVcfJobTest() throws JobExecutionException, IOException, URI
                 .build()
         );
 
-        assertEquals(EvaPipelineJobLauncherCommandLineRunner.EXIT_WITHOUT_ERRORS,
-                evaPipelineJobLauncherCommandLineRunner.getExitCode());
+        assertEquals(EXIT_WITHOUT_ERRORS, evaPipelineJobLauncherCommandLineRunner.getExitCode());
 
-        assertFalse(jobExplorer.getJobInstances(GENOTYPED_VCF_JOB, 0, 1).isEmpty());
-        JobExecution jobExecution = jobExplorer.getJobExecution(jobExplorer.getJobInstances(GENOTYPED_VCF_JOB, 0, 1)
-                .get(0).getInstanceId());
+        JobExecution jobExecution = getLastJobExecution(GENOTYPED_VCF_JOB);
+        assertCompleted(jobExecution);
```

Comment:
This method could be in a utility class, because many classes are performing these checks (for completed, failed or whatever combination). Please make it consistent across the whole codebase, either using the method everywhere or going back to the 2 asserts in this class.